CalderaWP/Caldera-Forms
# Impliment basic form cache

Fixes #3553 

## What Has Changed

* When querying for form, I put form in a variable, use that instead of query if needed.
* Tests showing that duplicate queries are not run.
* Specifics detailed here: https://github.com/CalderaWP/Caldera-Forms/pull/3566#issuecomment-648293099

## How To Test

* Make sure tests pass
* Check in wp-admin, page without forms and page with form for duplicate queries using Query Monitor plugin
